libs/api/src/lib/graphql/utils.ts

```diff
@@ -1,8 +1,17 @@
 import {Prisma} from '@prisma/client'
 import {DateFilterComparison} from '../db/common'
+import {AuthSession, AuthSessionType} from '@wepublish/authentication/api'
 
 export const mapDateFilterToPrisma = (
   comparison: DateFilterComparison
 ): keyof Prisma.DateTimeFilter => {
   return comparison === DateFilterComparison.Equal ? 'equals' : comparison
 }
+
+type IsMeBySessionArgs = {
+  id: string
+  session?: AuthSession
+}
+
+export const isMeBySession = ({id, session}: IsMeBySessionArgs) =>
```

Comment:
can we use multiple args instead of an object here? Doesn't improve anything here